### PR TITLE
udpate wait_http_test method to take verfiy paramater for verifying t…

### PR DESCRIPTION
**Pull request by RafyAmgadBenjamin**
I have updated  `wait_http_test`  and `check_url_reachable` to take `verify` param to be to use these methods against endpoint without SSL

### CODE REVIEW

File: jumpscale/sals/nettools/__init__.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def wait_connection_test(ipaddr: str, port: int, timeout=5):
             return True
 
 
-def wait_http_test(url: str, timeout: int = 60) -> bool:
+def wait_http_test(url: str, timeout: int = 60, verify: bool = True) -> bool:
```
Comment (xmonader) [inline jumpscale/sals/nettools/__init__.py:98]: Please update the usage of verify param in the docstring of the functionn

File: jumpscale/sals/nettools/__init__.py [hunk 2]
```diff
@@ -117,7 +117,7 @@ def wait_http_test(url: str, timeout: int = 60) -> bool:
         return False
 
 
-def check_url_reachabel(url: str, timeout=5):
+def check_url_reachabel(url: str, timeout=5, verify=True):
```
Comment (xmonader) [inline jumpscale/sals/nettools/__init__.py:120]: typo: should `reachable`

File: jumpscale/sals/nettools/__init__.py [hunk 3]
```diff
@@ -117,7 +117,7 @@ def wait_http_test(url: str, timeout: int = 60) -> bool:
...
     """Check that given url is reachable
 
     Args:
```
Comment (xmonader) [inline jumpscale/sals/nettools/__init__.py:123]: please document the verify param

**Review by xmonader — CHANGES_REQUESTED** [review]
Added minor comments to address, typos and docstrings.

File: jumpscale/sals/nettools/__init__.py (hunk 3 above)
Comment (RafyAmgadBenjamin) [reply jumpscale/sals/nettools/__init__.py:123]: done

File: jumpscale/sals/nettools/__init__.py (hunk 1 above)
Comment (RafyAmgadBenjamin) [reply jumpscale/sals/nettools/__init__.py:98]: done

File: jumpscale/sals/nettools/__init__.py (hunk 2 above)
Comment (RafyAmgadBenjamin) [reply jumpscale/sals/nettools/__init__.py:120]: done, the typo was there already

**Review by OmarElawady — APPROVED** [review]

**Review by OmarElawady — APPROVED** [review]

**Review by xmonader — APPROVED** [review]

**Pull request merged**